webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -1347,69 +1348,6 @@ def mocked_get(url, params, **options):
         ok_(dump['total'])
 
     @mock.patch('requests.get')
-    def test_carry_mware_error_codes(self, rget):
```

Comment:
Don't we have that already?
Perhaps not. I'll take a look. NB, this is specifically for the web api.